index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>InnovationX Africa Summit</title>
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css"/>
+        <link rel="stylesheet" href="./style/style.css">     
+    </head>
+    <body>
+      <section class="container">
+        <header>
```

Comment:
- Hey, please, let's try to align the top and bottom nav in the header of the desktop version to be on the same vertical line as seen in the images below to improve our design to be close to the main design as much as we can below:

*your design*
![Screenshot from 2023-04-29 19-53-20](https://user-images.githubusercontent.com/59397159/235321100-3c81be04-2195-46be-9ed4-728470ab7d1b.png)

*main design*
![Screenshot from 2023-04-29 20-38-42](https://user-images.githubusercontent.com/59397159/235321195-c2ed8ad4-540b-43e3-aa9d-6aa4a27749f1.png)